src/charm.py

```diff
@@ -255,28 +277,23 @@ def _configure(self, event: Optional[ConfigChangedEvent] = None) -> None:  # noq
             )
 
     def _on_remove(self, event: RemoveEvent):
-        """Handle remove charm event.
+        """Handle certs removed event.
```

Comment:
This doesn't seem right. Isn't this listening to the event triggered by Juju on removal?

Reply:
good catch, this is very old. reverted.